ansible/roles/osdsdb/scenarios/container.yml

```diff
@@ -18,10 +18,16 @@
     name: docker-py
 - name: run etcd containerized service
   docker_container:
-    name: myetcd
+    name: soda-etcd
     image: "{{ etcd_docker_image }}"
-    command: /usr/local/bin/etcd --advertise-client-urls http://{{ etcd_host }}:{{ etcd_port }} --listen-client-urls http://{{ etcd_host }}:{{ etcd_port }} --listen-peer-urls http://{{ etcd_host }}:{{ etcd_peer_port }}
     state: started
+    restart_policy: always
     network_mode: host
     volumes:
     - "/usr/share/ca-certificates/:/etc/ssl/certs"
+    - "etcd_data:/var/lib/etcd/"
```

Comment:
IMHO, The PR do not have any changes related to "etcd_data" volume creation.

Reply:
So while testing this step created the etcd_data volume and also retained it and finally removed it.

I looked around and seems like Docker will create a named volume if it doesn't exist when provided as part of the mount list.

Check [this comment](https://stackoverflow.com/a/41807606) on a stackoverflow.com question that I found.

Reply:
@AmitRoushan do you mean that this PR is related to restart policy only and should be separated with data vol creation for etcd?

Reply:
@kumarashit As I understand docker volumes need to be created before being assigned to the container. 
In the line I had assigned the volume directly to the container. As explained and  per the link above named volumes can be assigned directly.
Considering the data volume needs to be part of the container creation IMO the data volume should go with this PR.
We can close any related issues as well.

Reply:
@kumarashit The comment was to create volume before assigning it to container. But ansible+docker support named volume creation with "volume" attribute itself. The comment is no longer valid.